src/traits/window.rs

```diff
@@ -6,8 +6,21 @@ use glib::translate::{from_glib_none, ToGlibPtr};
 use ffi;
 use glib::to_gboolean;
 use cast::GTK_WINDOW;
+use gdk;
 
 pub trait WindowTrait : ::WidgetTrait {
+    fn move_(&self, x: i32, y: i32) {
```

Comment:
But `c_int` is an `i32`: http://doc.rust-lang.org/libc/libc/types/os/arch/c95/type.c_int.html